opentreemap/manage_treemap/templates/manage_treemap/partials/user_roles.html

```diff
@@ -19,6 +19,7 @@ <h3 class="management-header">{% trans "Invited tree map users" %}</h3>
                 <th data-sort>
                     <a href="{{ invite_sort_url }}&invite_sort={{ 'admin'|reverse_if_current:invite_sort }}">{% trans "Management Access" %}</a>
                 </th>
+                <th data-class="edit">Action</th>
```

Comment:
We should wrap this with `trans`.

I can make this column disappear with the following steps:

- Go to the users management page
- Click "Edit"
- Click "Cancel"